packages/console/src/components/AppBoundary/index.tsx

```diff
@@ -12,13 +11,13 @@ type Props = {
 };
 
 const AppBoundary = ({ children }: Props) => {
-  const defaultTheme = localStorage.getItem(themeStorageKey) ?? AppearanceMode.SyncWithSystem;
-  const { configs } = useAdminConsoleConfigs();
-  const theme = configs?.appearanceMode ?? defaultTheme;
+  const {
+    data: { appearanceMode, language },
+  } = useUserPreferences();
 
   useEffect(() => {
-    const isFollowSystem = theme === AppearanceMode.SyncWithSystem;
-    const className = styles[theme] ?? '';
+    const isFollowSystem = appearanceMode === AppearanceMode.SyncWithSystem;
```

Comment:
nit: better name with the past of the verb or align with `syncWithSystem` → `followedSystem` or `isSyncWithSystem`？

Reply:
Cool